lib/commands/sdk.js

```diff
@@ -446,6 +446,11 @@ SdkSubcommands.select = {
 					logger.error(__('Invalid Titanium SDK "%s"', value));
 					return false;
 				}
+				if(parseInt(value.split('.').slice(0,3).join('')) < 320) {
+					console.log('');
+					logger.warn(__('Selected SDK contains deprecated environment detection code.' +
+						'\n        Please choose a 3.2 or newer SDK for more accurate information.'));
```

Comment:
Please split this up into 2 separate `logger.warn()` calls. When we translate strings, we do it one sentence at a time. This means you no longer need the `\n` and all the whitespace before "Please".